modal/_container_entrypoint.py

```diff
@@ -489,21 +465,19 @@ def run_inputs(input_id, args, kwargs):
                         raise InvalidError(f"Generator function returned value of type {type(res)}")
 
                     for value in res:
-                        function_io_manager.enqueue_generator_value(
+                        function_io_manager.push_generator_value(
                             input_id, started_at, output_index.value, value, imp_fun.data_format
                         )
                         output_index.increase()
```

Comment:
This part could probably be slightly more efficient as a batched request if it were a single RPC over UDS, but that's okay. We can always optimize later if it's actually an issue!